FEM/rf_random_walk.cpp

```diff
@@ -5980,8 +6026,9 @@ void DATWriteParticleFile(int current_time_step)
 	sprintf(now, "%i", current_time_step);
 	string nowstr = now;
 
-	string vtk_file_name =  pathJoin(defaultOutputPath, FileName);
-	vtk_file_name += "_RWPT_" + nowstr + ".particles.vtk";
+	string vtk_file_name = FileName + "_RWPT_";
```

Comment:
ctests run and files are compared. However, the result, 3d_grain_RWPT_20.particles.vtk, is different from its reference.